src/components/TranscationLabelStatus.tsx

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react-native/no-unused-styles */
+import React from 'react';
+import { StyleSheet } from 'react-native';
+
+import { TxType } from 'app/consts';
+import { palette } from 'app/styles';
+
+import { Label } from './Label';
+
+const i18n = require('../../loc');
+
+const readableTransactionType = {
+  [TxType.ALERT_PENDING]: i18n.transactions.label.pending,
+  [TxType.ALERT_RECOVERED]: i18n.transactions.label.cancelled,
+  [TxType.ALERT_CONFIRMED]: i18n.transactions.label.done,
+  [TxType.RECOVERY]: i18n.transactions.label.recovered,
+};
+
+export const TranscationLabelStatus = ({ type }: { type: TxType }) => (
```

Comment:
no need to pass additionalLabelStyle (but it can stay in Label component as it is always nice additional option). Use switch case and the label with proper type 
<img width="610" alt="Zrzut ekranu 2020-08-3 o 12 00 11" src="https://user-images.githubusercontent.com/67273035/89171213-1f3fe500-d581-11ea-9aa0-c51dfb357537.png">

Reply:
what does it give us?

Reply:
it gives us that later if we will have another label with some text and with colors that "warning" label has we won't have to pass again color to it. It is more like we are using the "warning" label for ALERT_PENDING status, not the ALERT_PENDING defines the color for the label.

Reply:
It's an opinion after all but I would give +1 to object based assignment instead of switch case. It's much cleaner this way